src/containers/BooksList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+
+const BooksList = ({ books }) => (
+
+  <table>
+    <thead>
+      <tr>
+        <th>Book Id</th>
+        <th>Title</th>
+        <th>Category</th>
+      </tr>
+    </thead>
+    <Book books={books} />
```

Comment:
- [ ] <b>REQUIRED:</b> Refer to the previous comment, now you can apply the listing here:
```jsx
<table>
    <thead>
      <tr>
        <th>Book Id</th>
        <th>Title</th>
        <th>Category</th>
      </tr>
    </thead>
   { books.map((book) => (<Book book={book} />)) }
</table>
```